accurest-core/src/test/groovy/io/codearte/accurest/dsl/WiremockGroovyDslSpec.groovy

```diff
@@ -313,6 +315,168 @@ class WiremockGroovyDslSpec extends WiremockSpec {
     ''')
 	}
 
+	def "should generate request with urlPath and queryParameters for client side"() {
+		given:
+			GroovyDsl groovyDsl = GroovyDsl.make {
+				request {
+					method 'GET'
+					urlPath($(client("users"), server("items"))) {
+						queryParameters {
+							parameter 'limit': $(client(equalTo("20")), server(containing("10")))
```

Comment:
Makes sense, now I need to think about all possible combinations... I don't think there is a way how to check $(client(matching("reqex1")), server(matching("reqex2"))